### PR TITLE
Create new listing page.

### PR DESCRIPTION
**Pull request by zrideaux**
+ Set default site font family to sans serif.
+ Create newlisting.html and newlisting.css.
  + Vertical input form with preview at bottom.
  + Includes labels, sublabels, and input boxes.
image

### CODE REVIEW

File: src/main/webapp/newlisting.css [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+input[type=text],
+textarea {
+  background-color: #EEEEEE;
+  border: none;
+  border-radius: 15px;
+  max-width: 100%;
+  outline: none;
+  padding: 0.75em;
+  transition: 0.2s;
+  width: 100%;
```
Comment (kev1n80) [inline src/main/webapp/newlisting.css:42]: If the width is set to 100% I was wondering if max-width: 100% changes anything.

File: src/main/webapp/newlisting.html [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    <main>
+      <section class="shadow-box">
+        <h1>Create New Listing</h1>
+        <ul>
+          <li>
+            <label for="cause-name" class="form-label">A Name for the Cause</label>
+            <p class="sublabel">Give a short, descriptive name to the cause. This will be the first thing people see.</p>
+            <input id="cause-name" type="text">
+          </li>
+          <li>
```
Comment (kev1n80) [inline src/main/webapp/newlisting.html:26]: It seems like this list tag does not have a closing list tag

File: src/main/webapp/newlisting.html [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+        <div class="listing-info-container">
+          <div class="listing-heading-container">
+            <h2 id="listing-name">Listing Name</h2>
+            <div class="listing-tags-container">
+              <span class="listing-tag fundraiser-tag">Fundraiser</span>
+              <span class="listing-tag event-tag">Event</span>
+            </div>
+          </div>
+          <p>
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
```
Comment (kev1n80) [inline src/main/webapp/newlisting.html:93]: Please keep this line between 80-100 char long

File: src/main/webapp/newlisting.css (hunk 1 above)
Comment (zrideaux) [reply src/main/webapp/newlisting.css:42]: `max-width` prevents the resizable textarea from breaking page style! Unsure if there's a better way to do so

File: src/main/webapp/newlisting.html (hunk 2 above)
Comment (zrideaux) [reply src/main/webapp/newlisting.html:26]: Seems like it! Good catch

**Pull request merged**